src/testing/adaptive_sampling_testing.h

```diff
@@ -0,0 +1,54 @@
+#ifndef __ADAPTIVE_SAMPLING_TESTING__
+#define __ADAPTIVE_SAMPLING_TESTING__
+
+#include "tests.h"
+#include <fstream>
+#include <deal.II/numerics/solution_transfer.h>
+#include <deal.II/base/numbers.h>
+#include "parameters/all_parameters.h"
+#include "dg/dg_factory.hpp"
+#include "ode_solver/ode_solver_factory.h"
+#include "flow_solver/flow_solver.h"
+#include "flow_solver/flow_solver_factory.h"
+#include "pod_adaptive_sampling.h"
+#include "dg/dg.h"
+#include "parameters/all_parameters.h"
+#include "functional/functional.h"
+#include <eigen/Eigen/Dense>
+#include "functional/lift_drag.hpp"
```

Comment:
same comment as before about the includes

Reply:
Fixed in most recent commit.